src/person/person.repository.ts

```diff
@@ -19,7 +19,15 @@ export class PersonRepository extends RepositoryBase<IPerson> {
    */
   getPersonsByQuery(queryFields: any = {}, populate?: any, select?: any): Promise<IPerson[]> {
     const cond = {};
-    if (!(queryFields.alsoDead && queryFields.alsoDead === 'true')) cond['alive'] = 'true';
+
+    let query;
+    if (queryFields.status) {
+      query = queryFields.status.toLowerCase();
+    }
+    if (query !== consts.STATUS[3]) {
+      cond['status'] = consts.STATUS.indexOf(query) > -1 ? query : consts.STATUS[0];
```

Comment:
use `Array.includes()`

Reply:
there is some problem with includes() and es version used by kartoffel

Reply:
nvm